.vscode/cspell.json

```diff
@@ -17,7 +17,7 @@
     ".vscode/tasks.json",
     "NOTICE.txt",
     "eng/",
-    "*.dict.txt",
+    "**/.dict.txt",
```

Comment:
Good catch. I wonder if that's why "amqp" was an issue: cSpell currently (and probabl always will, per an issue conversation I was in with them) does not ignore its own configuration files. So it was very possible "amqp" was coming from inside the house.

Reply:
I don't know for sure - I noticed it because cspell triggered on a .dict.txt file on my local machine and while trying to figure it out, I noticed this.